charts/capsule/templates/pre-delete-job.yaml

```diff
@@ -1,4 +1,7 @@
-{{- $cmd := printf "kubectl scale deployment -n $NAMESPACE %s --replicas 0 &&" (include "capsule.deploymentName" .) -}}
+{{- $cmd := ""}}
+{{- if eq .Values.manager.kind "Deployment" }}
+{{- $cmd = printf "kubectl scale deployment -n $NAMESPACE %s --replicas 0 &&" (include "capsule.controllerName" .) -}}
```

Comment:
@prometherion sorry, not understand the reason to scale down before delete

Reply:
Not sure, I think we can drop it.

Reply:
It seems no effects, dropping it.